libs/shared/src/types/organization/index.ts

```diff
@@ -6,3 +8,8 @@ export enum ApiServiceLevelTypeEnum {
   // TODO: NV-3067 - Remove unlimited tier once all organizations have a service level
   UNLIMITED = 'unlimited',
 }
+
+export type IServiceApiRateLimits = Record<ApiServiceLevelTypeEnum, IApiRateLimits>;
```

Comment:
A map of API rate limits for a given service level. Used in the `get-default-api-rate-limit` use-case